### PR TITLE
Duel Creation

### PR DESCRIPTION
**Pull request by myugen**

### CODE REVIEW

File: services/match-service/domain/src/main/java/com/kjeldsen/match/domain/aggregate/Duel.java [hunk 1]
```diff
@@ -1,16 +1,50 @@
...
+                .result(duelResult)
+                .build();
+    }
+
+    public boolean isWin() {
+        return result == DuelResult.WIN;
+    }
+
+    public boolean isLose() {
+        return result == DuelResult.LOSE;
```
Comment (sheyko94) [inline services/match-service/domain/src/main/java/com/kjeldsen/match/domain/aggregate/Duel.java:48]: !isWin()

File: services/match-service/application/src/test/java/com/kjeldsen/match/application/usecases/PositionalDuelStartUseCaseTest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.kjeldsen.match.application.usecases;
+
+import org.junit.jupiter.api.Test;
+
+class PositionalDuelStartUseCaseTest {
+
+    @Test
+    void foo() {
+        // TODO: Implement test
```
Comment (sheyko94) [inline services/match-service/application/src/test/java/com/kjeldsen/match/application/usecases/PositionalDuelStartUseCaseTest.java:9]: Let's implement it before the merge

File: services/match-service/domain/src/main/java/com/kjeldsen/match/domain/aggregate/Duel.java (hunk 1 above)
Comment (myugen) [reply services/match-service/domain/src/main/java/com/kjeldsen/match/domain/aggregate/Duel.java:48]: Well, is more semantic, also, `!isWIn()` is valid for `result == DuelResult.LOSE` and `result == null`

Comment (sheyko94) [reply services/match-service/domain/src/main/java/com/kjeldsen/match/domain/aggregate/Duel.java:48]: convinced - good to go

**Pull request merged**